tests/test-cases/Group1-Docker-Commands/1-19-Docker-Volume-Create.robot

```diff
@@ -8,86 +8,62 @@ Suite Teardown  Cleanup VIC Appliance On Test Server
 Simple docker volume create
     ${rc}  ${output}=  Run And Return Rc And Output  docker ${params} volume create
     Should Be Equal As Integers  ${rc}  0
+    ${disk-size}=  Run  docker ${params} logs $(docker ${params} start $(docker ${params} create -v ${output}:/mydata busybox /bin/df -Ph) && sleep 10) | grep by-label | awk '{print $2}' 
+    Should Be Equal As Strings  ${disk-size}  975.9M
 
 Docker volume create named volume
     ${rc}  ${output}=  Run And Return Rc And Output  docker ${params} volume create --name=test
     Should Be Equal As Integers  ${rc}  0
     Should Be Equal As Strings  ${output}  test
+    ${disk-size}=  Run  docker ${params} logs $(docker ${params} start $(docker ${params} create -v ${output}:/mydata busybox /bin/df -Ph) && sleep 10) | grep by-label | awk '{print $2}' 
+    Should Be Equal As Strings  ${disk-size}  975.9M
 
 Docker volume create already named volume
-    ${status}=  Get State Of Github Issue  1562
-    Run Keyword If  '${status}' == 'closed'  Fail  Test 1-19-Docker-Volume-Create.robot needs to be updated now that Issue #1562 has been resolved
-    Log  Issue \#1562 is blocking implementation  WARN
-    #${rc}  ${output}=  Run And Return Rc And Output  docker ${params} volume create --name=test
-    #Should Be Equal As Integers  ${rc}  1
-    #Should Contain  ${output}  Error response from daemon: A volume named test already exists. Choose a different volume name.
+    ${rc}  ${output}=  Run And Return Rc And Output  docker ${params} volume create --name=test
```

Comment:
This test should really be independent of the other tests in case they fail for some reason. Can you do a volume create --name=testAlready followed by a volume create --name=testAlready?